### PR TITLE
OCaml solution

### PR DESCRIPTION
**Pull request by dce**
Borrowed the idea to include the subtractive values in the lookup table from @tommymarshall -- made the problem about 100x easier to solve than what I did in Haskell.

### CODE REVIEW

File: languages/OCaml/romanize.ml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+(* Return the highest Arabic/Roman pair less than or equal to number *)
+let highest_contained arabic = conversions
+                               |> List.filter ((>=) arabic % fst)
+                               |> List.last
+
+(* Recursively convert the Arabic number to a Roman numeral *)
+let rec romanize arabic = match arabic with
+  | 0 -> ""
+  | n -> let (a, r) = highest_contained n in r ^ romanize (n - a);;
```
Comment (nhunzaker) [inline languages/OCaml/romanize.ml:25]: Two semicolons?

Comment (nhunzaker) [inline languages/OCaml/romanize.ml:25]: And `^` is concat? neat

Comment (dce) [inline languages/OCaml/romanize.ml:25]: Yeah, `^` is string concat.

Comment (dce) [inline languages/OCaml/romanize.ml:25]: I can't find a coherent explanation for why you need the double semicolons, but you do. Something about the separation between simply defining things and code to be immediately executed. I may just be doing it wrong.

**Comment by nhunzaker** [comment]
Super cool, :+1:

**Pull request merged**